Expo1900/Expo1900/Extension/extension+Array.swift

```diff
@@ -7,11 +7,11 @@
 
 import Foundation
 
-extension Array where Element == Item {
-    subscript(safe index: Int) -> Item {
-        if self.count > index {
-            return self[index]
+extension Array {
+    subscript(safe index: Int) -> Element? {
+        if self.count <= index || Int.zero > index {
+            return nil
         }
```

Comment:
요건 약간 취향 차이이긴 한데.. 이렇게 로직의 앞단에서 조건을 체크하고, 넘어가지 못할 때 반환하는 방어로직 느낌의 로직은 `if`보단 `guard`가 더 이해하기 쉬운 것 같아요 🤔 
범위 체크 조건문도 `||` 연산자보다 `contains`를 활용하면 더 깔끔해질 것 같습니다.
(이건 주관적인 생각이니 참고만 해주세요 🙏 )
```suggestion
        guard (0..<count).contains(index) else {
            return nil
        }
```